build/root/WORKSPACE

```diff
@@ -1,8 +1,8 @@
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "441e560e947d8011f064bd7348d86940d6b6131ae7d7c4425a538e8d9f884274",
-    strip_prefix = "rules_go-c72631a220406c4fae276861ee286aaec82c5af2",
-    urls = ["https://github.com/bazelbuild/rules_go/archive/c72631a220406c4fae276861ee286aaec82c5af2.tar.gz"],
+    sha256 = "8e51b3d7902a0ed69e84ff7c30c2141d74970b8caa8b44bc2f6eded76f0abdef",
+    strip_prefix = "rules_go-3fd320621566c44aad0f7df5d0f5858efd2bbcf6",
```

Comment:
can you please revert these changes and instead cherry-pick in https://github.com/ixdy/kubernetes/commit/3854c3a6f9bfe4b502fa6910ad31751e3fdef88d?

Reply:
Done!